boards/feather_f405/board.h

```diff
@@ -0,0 +1,32 @@
+#ifndef BOARD_H
+#define BOARD_H
+
+#define OSC_FREQ 12
+#define USBDEVICESTRING "Feather F405"
+#define USBMFGSTRING "Adafruit"
+#define BOARD_FLASH_SECTORS 8
+#define BOARD_FLASH_SIZE (512 * 1024)
+
+#ifdef DEFINE_CONFIGDATA
+__attribute__((section(".config"))) __attribute__((used)) //
+const uint32_t configData[] = {
+    /* CF2 START */
+    513675505, 539130489, // magic
+    27, 100,  // used entries, total entries
+    204, 0x80000, // FLASH_BYTES = 0x80000
+    205, 0x20000, // RAM_BYTES = 0x20000
+    208, 0x69d51bac, // BOOTLOADER_BOARD_ID = 0x69d51bac
+    209, 0x57755a57, // UF2_FAMILY = STM32F401
+    210, 0x10, // PINS_PORT_SIZE = PA_16
```

Comment:
you could add feather pinout like here: https://github.com/microsoft/uf2-samdx1/blob/master/boards/arcade_feather_m4/board_config.h#L74 (edit this config file using https://microsoft.github.io/uf2/patcher/ )

Reply:
done!